commit d3232e22e981ffca763ed91bb73ee7a95939ed7a
Author: q66 <q66@chimera-linux.org>
Date:   Sun Jun 2 20:52:48 2024 +0200

    do not intercept egg info installation
    
    Doing so breaks compatibility with pep517 tools. The egg info
    is not installed when using pep517 tools anyway.

diff --git a/setup.py b/setup.py
index b45d315..cdb0be6 100755
--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,6 @@ import subprocess
 
 import setuptools
 import setuptools.command.install
-import setuptools.command.install_egg_info
 
 
 # distutils will be deprecated in python 3.12 in favor of setuptools,
@@ -233,14 +232,6 @@ from %(pkgname)s import %(filename)s
         distutils.command.build.build.run(self)
 
 
-class my_egg_info(setuptools.command.install_egg_info.install_egg_info):
-    """
-    Disable egg_info installation, seems pointless for a non-library
-    """
-    def run(self):
-        pass
-
-
 class my_install(setuptools.command.install.install):
     """
     Error if we weren't 'configure'd with the correct install prefix
@@ -538,7 +529,6 @@ setuptools.setup(
         'build_i18n': my_build_i18n,
 
         'install': my_install,
-        'install_egg_info': my_egg_info,
 
         'configure': configure,
 
